packages/@react-spectrum/cards/src/CardView.tsx

```diff
@@ -40,6 +42,15 @@ function CardView<T extends object>(props: SpectrumCardViewProps<T>, ref: DOMRef
   let collator = useCollator({usage: 'search', sensitivity: 'base'});
   let isLoading = loadingState === 'loading' || loadingState === 'loadingMore';
   let cardViewLayout = useMemo(() => typeof layout === 'function' ? new layout({collator}) : layout, [layout, collator]);
+  let layoutType = cardViewLayout.layoutType;
+
+  if (typeof layout === 'function') {
+    if (layoutType === 'grid') {
+      cardViewLayout.itemPadding = scale === 'large' ? 116 : 95;
+    } else if (layoutType === 'gallery') {
+      cardViewLayout.itemPadding = scale === 'large' ? 143 : 114;
+    }
+  }
```

Comment:
If user passes a layout constructor to the CardView we have set the itemPadding for the user depending on the scale since the layout doesn't have a concept of scale. Perhaps the layout should have a scale property?